cypress-tests/cypress/fixtures/merchant-create-body.json

```diff
@@ -18,19 +18,23 @@
       "city": "San Francisco",
       "state": "California",
       "zip": "94122",
-      "country": "US"
+      "country": "US",
+      "first_name": "john",
+      "last_name": "Doe"
     }
   },
-  "return_url": "https://example.com",
+  "return_url": "https://google.com/success",
   "webhook_details": {
     "webhook_version": "1.0.1",
     "webhook_username": "ekart_retail",
     "webhook_password": "password_ekart@123",
+    "webhook_url": "https://integ.hyperswitch.io/api/webhooks/cypress_merchant_GHAction_6dcfed64-ec62-46a2-9961-cafa3c1faa01/authorizedotnet",
```

Comment:
why are these hard coded? webhooks in cypress, from what i'm aware, is not integrated.